packages/bundle-source/demo/trailing-comment.js

```diff
@@ -12,3 +12,5 @@ export function buildRootObject() {
 }
 // comment
 // */ fooBarBaz!
+
+// /*
```

Comment:
I'd be slightly more reassured if the comment-opener test were in a separate file, so it couldn't interact with the comment-closer test nearby.